### DISCUSSION TITLE
No detection logic for wxWidgets 3.1.7 without Conan

### DISCUSSION THREAD

**Discussion by bulvara** [audacity/audacity#3253] [Q&A]
I'm trying to compile against wxgtk3 3.1.7 on ArchLinux with [this recipe](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=audacity-systemlibs):

```
CMake Error at cmake-proxies/cmake-modules/AudacityDependencies.cmake:238 (message):
  Failed to find the system package wxWidgets
```

I would like to give a shit on Conan hence `-Daudacity_conan_enabled=false \` to ensure using only system dependencies. In this case there is no detection logic for wxWidgets 3.1!

**Comment by crsib** [comment]
Audacity builds few time a day against system packaged wxWidgets on Fedora. Given the amount of information provided looks like you are doing something wrong.

**Comment by bulvara** [comment]
https://github.com/audacity/audacity/blob/master/linux/packages/arch/PKGBUILD

Is the "audacity-offline-dependencies-arch-linux.tar.gz" file from https://github.com/audacity/audacity/blob/9df3c4b398a7a35dc710ccbf3d780fd83d4d52d7/linux/packages/arch/PKGBUILD#L51 still available somewhere?
